packages/language-server/src/core/index.ts

```diff
@@ -61,9 +61,9 @@ export function getLanguageModule(
 							module: ts.ModuleKind.ESNext ?? 99,
 							target: ts.ScriptTarget.ESNext ?? 99,
 							jsx: ts.JsxEmit.Preserve ?? 1,
```

Comment:
Keeping this one for now because I don't see a way to set it through a pragma. Since this is only for type checking, it should be fine to keep it at Preserve, I think